tests/e2e/test_e2e.py

```diff
@@ -326,3 +326,28 @@ def test_e2e_single_streaming():
     expected_values = [4.0, 8.0, 12.0]
     for i, output in enumerate(outputs):
         assert output["output"] == expected_values[i], f"Intermediate output {i} is not expected value"
+
+
+@e2e_from_file("tests/e2e/default_openai_embedding_spec.py")
```

Comment:
Another test with a little more coverage on the failure cases would help avoiding breakages in the future.

It's good to pad things in view of future changes, this is what tests are really for after all.